components/form-builder/store/useTemplateStore.tsx

```diff
@@ -377,6 +378,10 @@ const createTemplateStore = (initProps?: Partial<InitialTemplateStoreProps>) =>
             getIsPublished: () => get().isPublished,
             getName: () => get().name,
             getDeliveryOption: () => get().deliveryOption,
+            isDeliveryOptionVault: () => {
```

Comment:
I was looking elsewhere and the only way we've been checking if the delivery option is via email or sent to the vault is doing a logic check every single time (i.e. if it's set, it's an object containing the email info, if it's undefined that means it's sent to the vault). I was hoping to reduce errors by having a helper functionality in the store itself. 

I don't mind renaming it to `getIsDeliveryOption` but it sounds a little off considering it's not an actual data attribute in the model itself so we're not doing a get command to retrieve the value of an attribute. What do you think?

Reply:
@timarney any thoughts on including helper functions in the store vs separate hooks?

Reply:
What about adding a utility function where you can pass the value from the store vs adding the function to the store itself.

parseDeliveryOption(deliveryOption){
 // check for  emailAddress
 return DeliveryOption.email 
 
...
 
 return DeliveryOption.vault
 
}

This could be re-used in the /components/form-builder/app/SetResponseDelivery.tsx